pkg/auth/oidc/impl.go

```diff
@@ -24,38 +25,44 @@ import (
 )
 
 const (
-	cookieNamePrefix              = "authn_cookie"
+	cookieNamePrefix              = "cf_golib_oidc_authn_cookie"
 	secondsBetweenCleanup         = 60
 	cookieExpirationHours         = 3
 	maxAgeSecondsRedirCookie      = 120
-	redirCookieName               = "oauth2_redir"
+	redirCookieName               = "cf_golib_oidc_oauth2_redir"
 	oauth2redirectPath            = "/oauth2/redirectendpoint"
-	authNCookieExpirationDuration = 8 * 3600 * time.Second
+	authNCookieExpirationDuration = 12 * time.Hour
 )
 
 type accessToken struct {
 	AccessToken string `json:"access_token"`
 	TokenType   string `json:"token_type"`
-	ExpiresIn   int    `json:expires_in`
+	ExpiresIn   int    `json:"expires_in"`
```

Comment:
ExpiresAt ?

Reply:
`expires_in` is what Keymaster is sending.
Also, the only change I made is to put in the missing quotes, which otherwise prevent the JSON decoder from extracting this information.

Reply:
ack